### PR TITLE
adding how to  install  for linux users

### PR DESCRIPTION
**Pull request by abdellatif-temsamani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 # How to install
 
+## windows
```
Comment (bitbrain) [inline README.md]: can you make this PascalCase instead of lowercase, please?

File: README.md [hunk 2]
```diff
@@ -24,10 +25,28 @@
     interface/editor/code_font = "ABSOLUTE_PATH_TO_ YOUR_FONT/CascadiaMono.ttf"
     ```
 
+## linux
```
Comment (bitbrain) [inline README.md]: can you make this PascalCase instead of lowercase, please?

File: README.md [hunk 3]
```diff
@@ -24,10 +25,28 @@
...
+
+1. open terminal.
+
+2. intstall fonts with package manager.
```
Comment (bitbrain) [inline README.md]: change this to:
```
2. Install fonts with a package manager, **or**:
```

File: README.md [hunk 4]
```diff
@@ -24,10 +25,28 @@
...
+
+**or**.
+
+2. download the fonts and copy them to ~/.local/share/fonts/
```
Comment (bitbrain) [inline README.md]: Capitalise the word, so change it to `Download`

**Comment by bitbrain** [comment]
A couple of minor things - I will add contribution guidelines to make this more clear.

**Comment by abdellatif-temsamani** [comment]
i think it should be good now

**Comment by bitbrain** [comment]
Good job!

**Pull request merged**